hummingbot/client/command/config_command.py

```diff
@@ -402,6 +416,8 @@ def _get_config_var_with_key(key: str) -> ConfigVar:
             else:
                 await write_config_to_yml()
             self._notify(f"\nNew config saved:\n{key}: {str(value)}")
+            if isinstance(self.strategy, PureMarketMakingStrategyV2):
+                ConfigCommand.update_running_pure_mm(self.strategy, key, cv.value)
 
             if not self.config_complete:
                 choice = await self.app.prompt("Your configuration is incomplete. Would you like to proceed and "
```

Comment:
Use the text in the story description:
"Changing [parameter name] requires stopping the bot. Would you like to stop the bot and reconfigure?"

Reply:
I think we can change the question text in another task (for the upcoming config task). 
Right now, that prompt can be either from just `config` or `config single_key`. When we separate them, we'll have to revisit this part again.

Reply:
ok sounds good